examples/product-detail-page/src/components/ProductCard.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+  import { computed, defineProps } from "vue";
+  import {
+    getSmallestThumbnailUrl,
+    getSrcSetForMedia,
+  } from "@shopware-pwa/helpers-next";
+  import {
+    useAddToCart,
+    useProductPrice,
+    usePrice,
+  } from "@shopware-pwa/composables-next";
+
+  const props = defineProps(["product"]);
```

Comment:
Somehow this comment disappeared https://github.com/shopware/frontends/pull/149#discussion_r1169624755
Let's use proper `props` definitions :)

Reply:
och, the comment is for another component, we just missed this one :)